HonestHouse/HonestHouse/Sources/Presentation/Preset/View/PresetView.swift

```diff
@@ -133,79 +152,82 @@ struct PresetView: View {
     // Buttons
     private func addButtonView() -> some View {
         Button {
-            vm.send(action: .goToPresetDetail(.create, nil))
+            vm.send(action: .goToPresetEditor(.create, nil))
         } label: {
-            Image(systemName: "plus")
-                .font(.system(size: 24, weight: .semibold))
-                .foregroundStyle(Color.g12)
-                .frame(width: 50, height: 50)
-                .background(Circle().fill(Color.g0))
+            ZStack {
+                Circle()
+                    .fill(Color.g0)
+                    .frame(width: 50, height: 50)
+                    .shadow(color: .black, radius: 10)
+                Image(.plus)
+                    .resizable()
+                    .scaledToFit()
+                    .frame(width: 24, height: 24)
+            }
```

Comment:
```suggestion
           Image(.plus)
                    .resizable()
                    .scaledToFit()
                    .frame(width: 24, height: 24)
                    .padding(13)
                     .background(Color.g0)
                    .clipShape(Circle())
                    .shadow(color: .black, radius: 10)
```

ZStack으로 잡고 frame으로 깎아도 되고 이렇게 modifier로 정의해도 아마 될 겁니다...
막 성능상 큰 차이가 있는 건 아닌데 후자가 더 직관적입니당 ...
아니면 .overlay 쓰는 게 더 좋아요
ZStack은 모든 자식 뷰의 레이아웃 계산 필요해서 메모리를 더 쓴다고 해요

Reply:
와우 감사해요 숩 당장 적용해볼게요

Reply:
오호오~~~